docs/contributing/how-to-release.md

```diff
@@ -67,7 +69,15 @@ and then run it using the new DLLs.
     * Update `docs/msw/binaries.md`: at least the version, but possibly also
       the list of supported compilers.
 
-2. Commit the changes and tag the release using your GPG key:
+2. For micro relases, review the changelog to determine if any ABI changes have been made. If so, or if this is a major or minor release, update the C:R:A settings in build/tools/version.bkl. Then from the build/bakesfiles directory run
```

Comment:
```suggestion
2. For micro relases, review the changelog to determine if any ABI changes have been made. If so, or if this is a major or minor release, update the C:R:A settings in `build/tools/version.bkl`. Then from the `build/bakesfiles` directory run
```
Just to do it consistently, as it's already done for the other file/directory names in this file.